src/firebaseDb.ts

```diff
@@ -64,7 +64,7 @@ export const initiatilizeFirebaseInstance = async () => {
     } else if (isTestingEnvironment || isLocal) {
         console.log('\n🔥 Connecting to firebase emulator...')
         console.log('TIP: if it doesn\'t connect after 10 secs,' +
-            'make sure you have the firebase emulator running using the "yarn dev" command')
+            'make sure you have the firebase emulator running using the "yarn dev:startlocaldb" command')
```

Comment:
```suggestion
            ' make sure you have the firebase emulator running using the "yarn dev:startlocaldb" command')
```

Reply:
Nit: Need a space after the comma

![Screenshot 2023-09-20 at 3 53 27 PM](https://github.com/ourjapanlife/findadoc-server/assets/31802656/e7766a9e-2974-47b4-be4d-9ac04694155d)